src/lib.rs

```diff
@@ -91,6 +91,22 @@ fn state_desc(state: gstreamer::ffi::GstState) -> &'static str {
     }
 }
 
+fn query_name(query: &gstreamer::QueryRef) -> &'static str {
```

Comment:
Why not use the proper bindings API for this but instead go via FFI?

Reply:
We do not expose that API yet? The only reference I found was in our `fmt::Debug` implementation which is clearly not what I want here. Should we add it?

Reply:
Seems useful to have, yes

Reply:
Opened https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/merge_requests/1289 - I think this can be closed meanwhile.

Reply:
I think it is fine to keep this open and rebase later on. I don't consider open PRs to be a maintenance burden on repos with as low traffic as this.